src/test/java/io/github/acm19/aws/interceptor/http/AwsRequestSigningApacheInterceptorTest.java

```diff
@@ -156,17 +159,27 @@ void testGzipCompressedContent() throws Exception {
                 request.getFirstHeader("signedContentLength").getValue());
     }
 
-    private static class AddHeaderSigner implements Signer {
+    /**
```

Comment:
I'd remove all of these along with `<module name="JavadocVariable"/>` it's too verbose and unnecessary.